src/main/java/com/telenordigital/sms/smpp/pdu/BindResp.java

```diff
@@ -35,4 +39,33 @@ public static BindResp deserialize(final Command command, final ByteBuf buf) {
 
     return new BindResp(command, status, sequence, systemId, version);
   }
+
+  public String status() {
+    return Optional.ofNullable(BindRespStatus.ALL.get(commandStatus))
+        .map(BindRespStatus::toString)
+        .orElse("UNKNOWN[0x%02X]".formatted(commandStatus));
+  }
+
+  private enum BindRespStatus {
+    OK(0x00),
+    ALREADY_BOUND(0x05),
+    SYSTEM_ERROR(0x08),
+    BIND_FAILED(0x0d),
+    INVALID_PASSWORD(0x0e),
+    INVALID_SYSTEM_ID(0x0f);
+
```

Comment:
I've realized that adding a map like this has probably very little affect on performance, if any at all, due to the magic that JVM does with JIT compilation behind the scene. Just a note